xml/System.Runtime.InteropServices/CharSet.xml

```diff
@@ -94,7 +94,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>Marshal strings as multiple-byte character strings.</summary>
+        <summary>Marshal strings as multiple-byte character strings: the system default [Windows (ANSI) code page](/windows/win32/intl/code-pages) on Windows, and UTF-8 on Unix.</summary>
```

Comment:
On Windows it marshals to the MBCS of the current system culture code page, not the system default code page.

Reply:
Not according to the code I mentioned above. `CP_ACP` is specifically defined to be the system default. See the links to source above.

Reply:
Well I guess I'm wrong here then. When I was testing this a while back I must have figured out how to change the system default of my VM instead of the current culture. Sorry for the confusion.

Reply:
This is also a case where the documentation for these kind of APIs could be old/out of date. I am assuming the documentation is correct, I have _not_ verified the actual behavior of that argument.